app/src/main/java/de/danoeh/antennapod/fragment/ItemFragment.java

```diff
@@ -120,6 +120,7 @@ public static ItemFragment newInstance(long feeditem) {
     private View butAction2;
     private ItemActionButton actionButton1;
     private ItemActionButton actionButton2;
+    private View noMediaMessage;
```

Comment:
maybe name this `noMediaMessageView`

Reply:
Sorry for the late reply. @mohitshah3111999 and me recently came up with a naming convention for IDs. You can find it here: https://github.com/AntennaPod/AntennaPod/wiki/Code-style#resource-id-naming You can find the discussion in #4399

According to that convention, it should be `noMediaLabel`.